node/src/components/binary_port.rs

```diff
@@ -794,6 +797,57 @@ where
             };
             BinaryResponse::from_value(status, protocol_version)
         }
+        InformationRequest::Reward {
```

Comment:
Please add this request to the `binary_port_component()` test in `node/src/reactor/main_reactor/tests/binary_port.rs`

Reply:
added a test now